api/controllers/likeController.js

```diff
@@ -0,0 +1,34 @@
+const { likeService } = require('../services');
+const { CustomError } = require('../utils/error');
+
+const createLike = async (req, res) => {
+    try {
+        const { product_id } = req.body;
+        const user_id = req.user;
```

Comment:
```suggestion
        const { product_id: productId } = req.body;
        const userId = req.user;
```
javascript 내에서 변수명은 snake_case가 아닌 camelCase로 선언하는것이 컨벤션입니다.